pkgs/tools/security/regpg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, git
+, perl
+}:
+
+let
+  perlEnv = perl.withPackages (p: [ p.TextMarkdown ]);
+
+in
+stdenv.mkDerivation rec {
+  pname = "regpg";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner = "fanf2";
+    repo = "regpg";
+    rev = "regpg-${version}";
+    sha256 = "2ea99950804078190e1cc2a76d4740e3fdd5395a9043db3f3fe86bf2477d3a7d";
+  };
+
+  nativeBuildInputs = [ git perlEnv ];
+
+  postPatch = ''
+    patchShebangs ./
```

Comment:
This is slow over the entire source tree, try to narrow it down more.
(Also, no trailing `/` is needed)